app/src/main/java/org/oppia/android/app/player/exploration/ExplorationActivityPresenter.kt

```diff
@@ -101,6 +102,13 @@ class ExplorationActivityPresenter @Inject constructor(
       getExplorationFragment()?.handlePlayAudio()
     }
 
+    binding.actionBottomSheetOptionsMenu.setOnClickListener {
+      val bottomSheetOptionsMenu = BottomSheetOptionsMenu()
+      bottomSheetOptionsMenu.showNow(activity.supportFragmentManager, bottomSheetOptionsMenu.tag)
+    }
+
+    updateToolbarTitle(explorationId)
```

Comment:
I think this was accidentally put back after a conflict with the latest develop changes. This should be removed as it's being set below (after field initialization to avoid referencing uninitialized values).

Ditto for ``internalProfileId``--we no longer need that.

Reply:
I have fixed it @BenHenning